[docs] Improve the CDC DataStream API docs (#390)
ververica/flink-cdc-connectors: docs/content/about.md

  }
}
```
### Deserialization
The following JSON data show the change event in JSON format.

```json
{
  "before": {
    "id": 111,
    "name": "scooter",
    "description": "Big 2-wheel scooter",
    "weight": 5.18
  },
  "after": {
    "id": 111,
    "name": "scooter",
    "description": "Big 2-wheel scooter",
    "weight": 5.15
  },
  "source": {...},
  "op": "u",  // the operation type, "u" means this this is an update event 
  "ts_ms": 1589362330904,  // the time at which the connector processed the event
  "transaction": null
}
```
**Note:** Please refer [Debezium documentation](https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-events

> )  to know more the meaning of each field.

Author: ```suggestion
)  to know the meaning of each field.
```

  }
}
```

> The following JSON shows the basic CDC events. 

Reviewer: The following JSON data show the change event in JSON format.

```json
{
  "before": {
    "id": 111,
    "name": "scooter",
    "description": "Big 2-wheel scooter",
    "weight": 5.18
  },
  "after": {
    "id": 111,
    "name": "scooter",
    "description": "Big 2-wheel scooter",
    "weight": 5.15
  },
  "source": {...},
  "op": "u",
  "ts_ms": 1589362330904,
  "transaction": null
}
```

> Note: please refer to [Debezium documentation](https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-events
> )  about the meaning of each fields.

Reviewer: ```suggestion
**Note:** Please refer [Debezium documentation](https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-events
)  to know more the meaning of each field.
```

In some cases, users can use the `JsonDebeziumDeserializationSchema(true)` Constructor to enabled include schema in the message. Then the Debezium JSON message may look like this:
```json
{

>   "schema": {...},

Reviewer: How about giving some import fields  to make the include json data more different with the exclude json data ?

  "payload": {
    "before": {
      "id": 111,
      "name": "scooter",
      "description": "Big 2-wheel scooter",
      "weight": 5.18
    },
    "after": {
      "id": 111,
      "name": "scooter",
      "description": "Big 2-wheel scooter",
      "weight": 5.15
    },
    "source": {...},

>     "op": "u",
>     "ts_ms": 1589362330904,

Reviewer: we can add basic notes for key fields like "op", "ts_ms"

```suggestion
    "op": "u",  // the operation type, "u" means this this is an update event 
    "ts_ms": 1589362330904, // the time at which the connector processed the event.
```

    "transaction": null
  }
}
```

> Usually, this is not recommended to include schema because this makes the messages very verbose and reduces parsing performance.

Reviewer: ```suggestion
Usually, it is recommended to exclude schema because schema fields makes the messages very verbose which reduces parsing performance.
```
